redi-connect-front/src/pages/app/me/Mentoring.tsx

```diff
@@ -57,12 +55,14 @@ const Mentoring = ({ profile, profileSaveStart }: any) => {
 
   const { categories: selectedCategories } = formik.values
 
+  const categoryObject = Object.assign({}, ...availableCategories.map(category => ({ [category.id]: category.label })))
```

Comment:
I removed this line:
```
const categoryObject = Object.assign({}, ...availableCategories.map(category => ({ [category.id]: category.label })))
```

since this is already done in config.tsx:50 - categoriesIdToLabelMap